Data Generator.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+first_names = ["James", "Mary", "Liam", "Olivia", "Robert", "Patricia", "Noah", "Emma", "John", "Jennifer"]
+middle_names = ["Louise", "Rose", "Grace", "William", "Jane", "Elizabeth", "Thomas", "Anne", "Ann", "David"]
+last_names = ["Smith", "Johnson", "Williams", "Brown", "Jones", "Garcia", "Miller", "Davis", "Rodriguez", "Martinez"]
+
+names = []
+
+for i in range(10):
+    for j in range(10):
+        names.append(first_names[i] + ' ' + last_names[i])
+
+for i in range(10):
+    for j in range(10):
+        for k in range(10):
+            names.append(first_names[i] + ' ' + middle_names[j] + ' ' + last_names[k])
+
+countries = ["Switzerland", "Canada", "Sweden", "Australia", "United States", "Japan", "Germany", "New Zealand"]
+
+vaccines = [["Pfizer", "Mainz, Germany"], ["Moderna", "Massachusetts, U.S."], ["Novavax", "Maryland, U.S."]]
+
+random.shuffle(names)
+
+with open("E:/Vaccination DataBase/data_entry.tql", 'w') as f:

```

Comment:
I made the file opening location relative;
but I think that taking the file path from the user would not be feasible in this case, because this code is just inserting random data into the file, which the user should have no concern with, in the actual use case.